mailpoet/tests/integration/REST/Automation/Workflows/WorkflowPutEndpointTest.php

```diff
@@ -0,0 +1,138 @@
+<?php
+
+namespace MailPoet\REST\Automation\Workflows;
+
+use MailPoet\Automation\Engine\Builder\CreateWorkflowFromTemplateController;
+use MailPoet\Automation\Engine\Data\Step;
+use MailPoet\Automation\Engine\Data\Workflow;
+use MailPoet\Automation\Engine\Storage\WorkflowStorage;
+use MailPoet\REST\Automation\AutomationTest;
+
+require_once __DIR__ . '/../AutomationTest.php';
+
+class WorkflowPutEndpointTest extends AutomationTest
+{
+  private const ENDPOINT_PATH = '/mailpoet/v1/automation/workflows/%d';
+
+  /** @var WorkflowStorage */
+  private $workflowStorage;
+
+  /** @var CreateWorkflowFromTemplateController */
+  private $createWorkflow;
+
+  /** @var Workflow */
+  private $workflow;
+
+  public function _before() {
+    parent::_before();
+    $this->workflowStorage = $this->diContainer->get(WorkflowStorage::class);
+    $this->createWorkflow = $this->diContainer->get(CreateWorkflowFromTemplateController::class);
+    $this->workflow = $this->createWorkflow->createWorkflow('simple-welcome-email');
+    assert($this->workflow instanceof Workflow);
```

Comment:
I've seen multiple `assert(... instanceof ...)` throughout this PR. Could we use `$this->assertInstanceOf(...)` instead?

The reason is that we may add a sniff to ban `assert` in the future because in the `lib` code, `assert`s are marked as dangerous by some security plugins. And since our sniffs for the `lib` and `tests` are the same, this could cause some problems.